### DISCUSSION TITLE
Simple question

### DISCUSSION THREAD

**Discussion by papa-pitufo** [staxrip/staxrip#734] [General]
Sorry, but I would like to know.

Do you think to release the new V2.7 before vacation or after (On September, probably).?

I am not impatient, it is simple curiosity, at this time the programmers usually slow down the releases.

Best regards

**Comment by Dendraspis** [comment]
The next release (v2.7.0) is planned to be released within the next week.
@JJKylee made a lot of magnificent changes to the filters and he wants to make sure everything works, so a little bit of testing and double-checking is required.  🙂

**Comment by papa-pitufo** [comment]
Ok, thanks a lot.

And about test, obviusly, of course, every test you need .

Thnaks again
